### PR TITLE
Remove docs from this repo and use fluxible's docs folder

### PR DESCRIPTION
**Pull request by mridgway**
So many changes...

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: components/Doc.jsx [hunk 1]
```diff
@@ -15,13 +15,16 @@ function isModifiedEvent (e) {
 }
 
 var Component = React.createClass({
+    contextTypes: {
```
Comment (redonkulus) [inline components/Doc.jsx]: DOCS_URL above needs to be changed to the fluxible repo

Comment (redonkulus) [inline components/Doc.jsx]: Also I think the link in general is broken now too, please review.

File: package.json [hunk 2]
```diff
@@ -55,6 +55,7 @@
     "highlight.js": "^8.4.0",
     "marked": "^0.3.3",
     "react": "^0.13.0",
+    "routr": "^0.1.0",
```
Comment (redonkulus) [inline package.json]: Can remove the walk dependency now?

**Comment by redonkulus** [comment]
I'm going to merge and make the changes locally here to get this in.

**Pull request merged**